homeassistant/components/homewizard/strings.json

```diff
@@ -20,7 +20,7 @@
       }
     },
     "error": {
-      "api_not_enabled": "The API is not enabled. Enable API in the HomeWizard Energy App under settings",
+      "api_not_enabled": "[%key:component::homewizard::config::step::reauth_confirm::description%]",
```

Comment:
I don't necessarily think it's the same message for a description for a reauth flow and an error message. Perhaps it just seem to work here but in other languages it might be strange.